packages/next/package.json

```diff
@@ -82,6 +81,7 @@
     "loader-utils": "1.2.3",
     "mkdirp": "0.5.1",
     "next-server": "8.1.1-canary.54",
+    "node-libs-browser": "2.2.0",
```

Comment:
Is this dependency required here?

Reply:
Yes, this was the fix for the browser libraries, by retaining them as externals of the build.